mkosi

```diff
@@ -3501,6 +3500,7 @@ def build_stuff(args):
     # always the same
     args.machine_id = uuid.uuid4().hex
 
+    make_output_dir(args)
```

Comment:
We have this:

```python
    if args.cache_path is not None:
        args.cache_path = os.path.abspath(args.cache_path)
```

And handling the case it's None by creating a temporary directory which gets mounted in the final image... And it looks for a `mkosi.cache` dir when there's no option passed (either as a command line or in the configuration).

I couldn't find a case in which the cache is inside the the output dir, so I'm a little confused about what you are fixing. Could you share the steps to reproduce or backtrace ?

Reply:
With the regular mkosi.default that is in the repo, if I just do something like:

$ sudo ./mkosi -O images

And the "images" directory doesn't exist yet, then it will fail. I can open an issue with the proper backtrace if you don't see the same failure when running that command.

Reply:
i'd like to understand this better, tbh, which call precisely will fail?

Reply:
I moved this commit to #277 with a better commit message